addons/core/translations/resources/en-us.yaml

```diff
@@ -1238,13 +1238,40 @@ app-token:
   description: App tokens are long-lived, fine grained access tokens for authorization/authentication.
   titles:
     new: New App Token
+  tabs:
+    permissions: Permissions
+  about:
+    title: About this token
+  ttl:
+    title: Time to live (TTL)
+  tts:
+    title: Time to stale (TTS)
```

Comment:
thoughts on moving these translations labeled `title` to the `form` section? Since in the "New App Token" form they are fields as well with helper text.
```
form:
...
  ttl:
    label:
      0: Time to live (TTL)
      1: TTL
  tts:
    label:
      0: Time to stale (TTS)
      1: TTS
  permissions:
    label: Permissions
```